src/types.rs

```diff
@@ -54,6 +54,33 @@ impl BERDecodable for RsaPkcs15<Sha256> {
     }
 }
 
+pub struct EcdsaX962<H>(pub H);
+
+impl<H> SignatureAlgorithm for EcdsaX962<H> {}
+
+/// ecdsaWithSHA256
+impl DerWrite for EcdsaX962<Sha256> {
+    fn write(&self, writer: DERWriter) {
+        writer.write_sequence(|writer| {
+            writer.next().write_oid(&oid::ecdsaWithSHA256);
```

Comment:
For future reference:
* https://datatracker.ietf.org/doc/html/rfc5480#page-17
* https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.1.2